scripts/zones/Abyssea-Altepa/IDs.lua

```diff
@@ -71,28 +71,28 @@ zones[xi.zone.ABYSSEA_ALTEPA] =
     },
     mob =
     {
-        RANI_1                      = 17670551,
-        ORTHUS_1                    = 17670552,
-        DRAGUA_1                    = 17670553,
-        BENNU_1                     = 17670554,
-        RANI_2                      = 17670555,
-        ORTHUS_2                    = 17670556,
-        DRAGUA_2                    = 17670557,
-        BENNU_2                     = 17670558,
-        RANI_3                      = 17670559,
-        ORTHUS_3                    = 17670560,
-        DRAGUA_3                    = 17670561,
-        BENNU_3                     = 17670562,
-        IRONCLAD_SMITER             = 17670565,
-        AMAROK                      = 17670567,
-        SHAULA                      = 17670570,
-        EMPERADOR_DE_ALTEPA         = 17670571,
-        TABLILLA                    = 17670572,
-        SHARABHA                    = 17670574,
-        WAUGYL                      = 17670576,
-        CHICKCHARNEY                = 17670577,
-        VADLEANY                    = 17670578,
-        BUGUL_NOZ                   = 17670579,
+        RANI_1              = GetFirstID("Rani"),
+        ORTHUS_1            = GetFirstID("Orthrus"),
+        DRAGUA_1            = GetFirstID("Dragua"),
+        BENNU_1             = GetFirstID("Bennu"),
```

Comment:
The amount of offsets here doesn't really solve, but rather hides the problem.  There probably should be a better way to address these since there are multiple instances of the same mob.  Not going to comment on every instance, but + 201 below?  That's obscene, and there must be a better way.

Reply:
Maybe a GetSecondID / GetThirdID? Unless we can make something like GetTableIDs that would return the list of same IDs and pick the number you want instead of returning a full list.

Reply:
There are also instances of where placeholder of mobs are just all over the place with IDs. We would need a way to target a specific mob in a list with multiple names.

Reply:
I can make a GetSecondID and GetThirdID:

Prints the correct values instead of using + 201/etc.

```
KUKULKAN_1              = GetFirstID("Kukulkan"),
KUKULKAN_2              = GetSecondID("Kukulkan"),
KUKULKAN_3              = GetThirdID("Kukulkan"),
```
![image](https://github.com/LandSandBoat/server/assets/105882754/2b34afbd-63df-4f03-98cd-79c9b9e805a3)

Reply:
GetTableOfIDs(name)[1], etc?

Reply:
> GetTableOfIDs(name)[1], etc?

yes please

Reply:
Going back into my hole now.